Samples/Widgets/cpp-win32-packaged/SampleWidgetProviderApp/WidgetImplBase.h

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+class WidgetImplBase
+{
+public:
+    WidgetImplBase() {};
+
+    winrt::hstring Id() { return m_id; };
+    winrt::hstring State() { return m_state; };
+    bool IsActivated() { return m_isActivated; };
```

Comment:
nit, these can be marked as const and noexcept:

```suggestion
    winrt::hstring Id() const noexcept { return m_id; };
    winrt::hstring State() const noexcept { return m_state; };
    bool IsActivated() const noexcept { return m_isActivated; };
```